pkg/apiserver/apiserver.go

```diff
@@ -268,8 +268,6 @@ func (g *APIGroupVersion) InstallREST(container *restful.Container, root string,
 
 	// TODO: port the rest of these. Sadly, if we don't, we'll have inconsistent
 	// API behavior, as well as lack of documentation
-	mux := container.ServeMux
```

Comment:
Can you explain this change?  Passing the outer mux, rather than the inner?  Is this just because go-restful requires a ServeMux?  Maybe we should see if they'll change it to an interface.